src/Radio.js

```diff
@@ -37,8 +38,7 @@ const RadioWrap = styled.span`
 `
 
 const RadioInput = styled(Input)`
-  border-style: none;
-  display: inline;
+  display: none;
```

Comment:
Setting `display: none` on form elements creates accessibility issues – there are techniques for visibly hiding elements in an accessible way, which we should do here. See @broox9's PR for an example: https://github.com/pricelinelabs/design-system/pull/146/files#diff-4fff159aacbfb02718d812c3eddff067R45

Reply:
yeah you're right! I think Input already has `appearance: none` so we can just get rid of this